src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/nuget.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need a specific config for tool-runtime? If so plese remove the coreclr and corefxtestdata feeds as they shouldn't be needed.